pkg/Microsoft.NETCore.Platforms/frameworks.json

```diff
@@ -0,0 +1,84 @@
+{
+  "frameworkDefinitions": {
+    ".NETPlatform": {
+      "folderName": "dotnet",
+      "displayName": ".NET Platform",
+      "versions": {
+        "v5.1": {
+          "name": ".NETPlatform,Version=v5.1",
+          "folderName": "dotnet5.1",
+          "displayName": ".NET Platform 5.1"
+        },
+        "v5.2": {
+          "name": ".NETPlatform,Version=v5.2",
+          "folderName": "dotnet5.2",
+          "displayName": ".NET Platform 5.2"
+        },
+        "v5.3": {
+          "name": ".NETPlatform,Version=v5.3",
+          "folderName": "dotnet5.3",
+          "displayName": ".NET Platform 5.3"
+        },
+        "v5.10": {
+          "name": ".NETPlatform,Version=v5.10",
+          "folderName": "dotnet5.10",
+          "displayName": ".NET Platform 5.10"
+        }
+    },
+
+    ".NETFramework": {
+      "folderName": "net",
```

Comment:
Sure, so long as it has sufficient semantics to define the associations.  We'd have to examine current nuget behavior to make sure it would match 1:1.